Sources/Common/DataModel/ITKHelper/index.js

```diff
@@ -0,0 +1,30 @@
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+
+/**
+ * Converts an itk.js image to a vtk.js image.
+ *
+ * Requires an itk.js image as input.
+ */
+function convertItkToVtkImage(itkImage, options) {
+  // create VTK image data
+  const imageData = vtkImageData.newInstance({
+    origin: itkImage.origin.slice(),
+    spacing: itkImage.spacing.slice(),
+  });
+  const scalars = vtkDataArray.newInstance({
+    name: options.scalarArrayName || 'Scalars',
+    values: itkImage.data,
```

Comment:
no need to `slice()` data?

Reply:
I'd rather not make another copy of the image data, and ideally nobody else tries to modify it anyways.